xchange-bitso/src/main/java/org/knowm/xchange/bitso/dto/trade/BitsoOrderSide.java

```diff
@@ -0,0 +1,31 @@
+package org.knowm.xchange.bitso.dto.trade;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/** Order side enum */
+public enum BitsoOrderSide {
```

Comment:
you can keep the enums like that compacter just by using `@JsonProperty` for each value